lib/execution_engine2/db/MongoUtil.py

```diff
@@ -164,6 +190,10 @@ def get_job_log(self, job_id: str = None) -> JobLog:
 
         return job_log
 
+    def job_log_exists(self, job_id):
```

Comment:
is this used somewhere?

Reply:
no want me to remove it? thought it might be useful somewhere down the road.

Reply:
removed.